src/main/java/com/google/maps/model/PlaceIdScope.java

```diff
@@ -0,0 +1,15 @@
+package com.google.maps.model;
+
+/**
+ * Place ID scope is the scope of a Place ID returned from the Google Places API Web Service.
+ */
+public enum PlaceIdScope {
+  /**
+   * APP indicates the place ID is recognised by your application only.
```

Comment:
I think it's covered pretty well in the docs so wouldn't worry too much. Maybe just copy the rest of the doc description across?